src/completion/pass.bash-completion

```diff
@@ -59,7 +59,7 @@ _pass_complete_entries () {
 }
 
 _pass_complete_folders () {
-	local prefix="${PASSWORD_STORE_DIR:-$HOME/.password-store/}"
+	local prefix="${PASSWORD_STORE_DIR:-$HOME/.passage/store/}"
```

Comment:
Sorry, but I think it needs one more tweak: `local prefix="${PASSAGE_DIR:-$HOME/.passage/store/}"`. I think I left in the `PASSWORD_STORE_DIR` in my first version of the comment. My mistake.

Reply:
Updated.